consent_manager/examples/api_full_example.rst

```diff
@@ -0,0 +1,352 @@
+<!DOCTYPE html>
```

Comment:
Is this an error? Why this RST file contains HTML? Build fails on it.

Reply:
OK, i see that it should be included as example. I'll fix that.

Reply:
strange, our script works here,
this is included with 
```
.. literalinclude:: examples/api_full_example.rst
    :language: html
```